### PR TITLE
Rancher-EBS validation tests

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
@soumyalj This is ready to review and test.

**Comment by StrongMonkey** [comment]
This require two hosts environment in AWS. The host1 and host2 must be in the same availability zone.

File: tests/v2_validation/cattlevalidationtest/core/test_ebs_volume.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    client.wait_success(client.delete(stack))
+    while True:
+        try:
+            client.delete(volume)
+            break
+        except Exception:
+            time.sleep(1)
+            pass
+
+
```
Comment (soumyalj) [inline tests/v2_validation/cattlevalidationtest/core/test_ebs_volume.py]: Can we add a step to check if we can write to the volume and read from it ?
Also a short comment on the purpose of the test case does would be helpful.

Comment (StrongMonkey) [inline tests/v2_validation/cattlevalidationtest/core/test_ebs_volume.py]: will add it

**Comment by soumyalj** [comment]
LGTM

**Comment by soumyalj** [comment]
LGTM

**Pull request merged**